swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -301,69 +398,42 @@ public type ErrorMsg record {|
 
 ## Run the service
 
-In the terminal, navigate to the `covid19` directory, and execute the command below to run the service package.
+Use the [**Run**](/learn/vs-code-extension/run-a-program/) CodeLens of the VS Code extension to build and run the service, as shown below.
 
-> **Info**: The console should have warning logs related to the isolatedness of resources. It is a built-in service concurrency safety feature of Ballerina.
+![Run the service](/learn/images/featured-scenarios/write-a-restful-api-with-ballerina/run-the-service.gif)
 
+>**Info:** Alternatively, you can run this service by navigating to the project root (i.e., the `covid19` directory) and executing the `bal run` command. The console should have warning logs related to the isolatedness of resources. It is a built-in service concurrency safety feature of Ballerina.
 
-```
-$ bal run
-```
-
-You view the output below.
+You can view the output below in the Terminal.
 
 ```
 Compiling source
-	example/covid19:0.1.0
+	ballerina/covid19:0.1.0
 
 Running executable
 ```
 
 ## Try the service
 
-In another terminal, execute the cURL commands below one by one to try out the service.
+Use the [**Try it**](/learn/vs-code-extension/try-the-services/try-http-services/) CodeLens of the VS Code extension to send a request to the service to try out the use case.
 
 ### Get all countries
 
-Execute the cURL command below.
+Retrieve all the available records of all countries, as shown below.
 
-```
-$ curl http://localhost:9000/covid/status/countries
-```
-
-You view the output below.
-
-```
-[{"iso_code":"AFG", "country":"Afghanistan", "cases":159303, "deaths":7386, "recovered":146084, "active":5833}, {"iso_code":"SL", "country":"Sri Lanka", "cases":598536, "deaths":15243, "recovered":568637, "active":14656}, {"iso_code":"US", "country":"USA", "cases":69808350, "deaths":880976, "recovered":43892277, "active":25035097}]
-```
+![Get all countries](/learn/images/featured-scenarios/write-a-restful-api-with-ballerina/get-all-countries.gif)
 
 ### Add a country by the ISO code 
 
-Execute the cURL command below.
+Add a record of a country by its ISO code, as shown below.
 
```

Comment:
Let's give the snippet you enter as the payload here. Users can then just copy it and try it out.

Reply:
Added.